lib/views/frontend/spree/orders/_stripe_payment_request_button.html.erb

```diff
@@ -0,0 +1,48 @@
+<% if current_order.present? && cart_checkout_payment_method.present? %>
+  <div id="payment-request">
```

Comment:
can we add an HTML class here as well? it probably needs some style in the host application.